### ISSUE TITLE
Reference voice not working in web interface

### ISSUE THREAD

**Issue by h-eriksson** [SWivid/F5-TTS#667]
### Checks

- [X] This template is only for usage issues encountered.
- [X] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [X] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [X] I confirm that I am using English to submit this report in order to facilitate communication.

### Environment Details

Linux Mint 22, Conda 24.11.1, Python 3.10.16

### Steps to Reproduce

Followed installation by copy-pasting:
1. conda create -n f5-tts python=3.10
2. conda activate f5-tts
3. pip install torch==2.3.0+cu118 torchaudio==2.3.0+cu118 --extra-index-url https://download.pytorch.org/whl/cu118
4. pip install git+https://github.com/SWivid/F5-TTS.git

Started the web interface:
1. f5-tts_infer-gradio

### ✔️ Expected Behavior

When adding a sample sound into the Reference audio element, see a wave-form, and the length of seconds of the sample. 

### ❌ Actual Behavior

No waveform appears and the length is always 0:00. When clicking "Synthesize" only "Error" appears where the sound should be generated and displayed. No further details are displayed.

Added context: It did work for some time yesterday. Just recording into a mic and saving it from Audacity didn't work, so I used an online converter to get the same audio, but ensure better format if I was doing something wrong. These files worked for a bit. After a while, they stopped working either, both new ones, and old ones I already done and worked previously in the same session. The files are in 24khz, Mono, constant bit-rate and 16 kbps. I have tried with other settings such as 16khz as well, but still the same issue.
The samples are between 7-13 seconds in length.

This only happens in the web interface though. The cli-interface works:
f5-tts_infer-cli \
--model "F5-TTS" \
--ref_audio "/[my undisclosed home directory]/Voices/Reference voice.mp3" \
--ref_text "[Manually transcribed reference audio file]" \
--gen_text "This is some generated text that will be spoken by a reference voice."

This outputs the infer file cli_basic.wav in /tests/ with the correct voice that isn't accepted in the web interface.

I also want to say that I have been looking through the issues, but I couldn't find any subjects that looked relevant to my issue. If I missed some, or misunderstood the subject, I'm sorry.

**Comment by SWivid** [comment]
> No waveform appears and the length is always 0:00. When clicking "Synthesize" only "Error" appears where the sound should be generated and displayed. No further details are displayed.

will there be some output message at the commandline interface where you run f5-tts_infer-gradio

**Comment by h-eriksson** [comment]
No, nothing is happening in the terminal. This is the full and current output after I have added a voice unsuccessfully:

Building prefix dict from the default dictionary ...
Dumping model to file cache /tmp/jieba.cache
Loading model cost 0.785 seconds.
Prefix dict has been built successfully.
Word segmentation module jieba initialized.

Download Vocos from huggingface charactr/vocos-mel-24khz

vocab :  /[redacted path]/miniconda3/envs/f5-tts/lib/python3.10/site-packages/f5_tts/infer/examples/vocab.txt
token :  custom
model :  /[redacted path]/.cache/huggingface/hub/models--SWivid--F5-TTS/snapshots/4dcc16f297f2ff98a17b3726b16f5de5a5e45672/F5TTS_Base/model_1200000.safetensors 

/[redacted path]/miniconda3/envs/f5-tts/lib/python3.10/site-packages/gradio/components/chatbot.py:242: UserWarning: You have not specified a value for the `type` parameter. Defaulting to the 'tuples' format for chatbot messages, but this is deprecated and will be removed in a future version of Gradio. Please set type='messages' instead, which uses openai-style dictionaries with 'role' and 'content' keys.
  warnings.warn(
Starting app...
* Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.

**Comment by h-eriksson** [comment]
More information.
When I have uploaded a voice-sample, the terminal doesn't say anything, but when I try to generate something from that voice sample, I get a whole slew of logs. I followed the logs for a bit (together the champion log-reader ChatGPT so I won't have to read through 100s of lines).
We found the upload location of the temporary voice-files. They come out as 0 bytes large, so there's something going on with the upload. I haven't been able to figure out what yet though.
I did reinstall ffmpeg just be on the safe side (sudo apt purge ffmpeg / sudo apt install ffmpeg)

For now, when I have uploaded a file, I can simply replace it with the original. When I generate from that, it works perfectly fine.
